[2025-01-11T21:44:18.317+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-11T21:44:18.336+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dbt_dag.run_dbt_model manual__2025-01-11T21:44:17.245827+00:00 [queued]>
[2025-01-11T21:44:18.345+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dbt_dag.run_dbt_model manual__2025-01-11T21:44:17.245827+00:00 [queued]>
[2025-01-11T21:44:18.347+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-11T21:44:18.360+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): run_dbt_model> on 2025-01-11 21:44:17.245827+00:00
[2025-01-11T21:44:18.369+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dbt_dag', 'run_dbt_model', 'manual__2025-01-11T21:44:17.245827+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dbt_dag.py', '--cfg-path', '/tmp/tmp_ih2wul6']
[2025-01-11T21:44:18.372+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask run_dbt_model
[2025-01-11T21:44:18.375+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=252) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-11T21:44:18.377+0000] {standard_task_runner.py:72} INFO - Started process 253 to run task
[2025-01-11T21:44:18.425+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dbt_dag.run_dbt_model manual__2025-01-11T21:44:17.245827+00:00 [running]> on host c6530104b64d
[2025-01-11T21:44:18.521+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dbt_dag' AIRFLOW_CTX_TASK_ID='run_dbt_model' AIRFLOW_CTX_EXECUTION_DATE='2025-01-11T21:44:17.245827+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-11T21:44:17.245827+00:00'
[2025-01-11T21:44:18.522+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-11T21:44:22.447+0000] {logging_mixin.py:190} INFO - 21:44:20  Running with dbt=1.9.1
21:44:20  Registered adapter: postgres=1.9.0
21:44:21  Found 4 models, 2 seeds, 5 data tests, 2 sources, 546 macros
21:44:21  
21:44:21  Concurrency: 1 threads (target='dev')
21:44:21  
21:44:21  1 of 1 START sql table model jin_schema.my_first_dbt_model ..................... [RUN]
21:44:21  1 of 1 OK created sql table model jin_schema.my_first_dbt_model ................ [SELECT 2 in 0.13s]
21:44:21  
21:44:21  Finished running 1 table model in 0 hours 0 minutes and 0.23 seconds (0.23s).
21:44:21  
21:44:21  Completed successfully
21:44:21  
21:44:21  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-01-11T21:44:22.448+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-11T21:44:22.456+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-11T21:44:22.457+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dbt_dag, task_id=run_dbt_model, run_id=manual__2025-01-11T21:44:17.245827+00:00, execution_date=20250111T214417, start_date=20250111T214418, end_date=20250111T214422
[2025-01-11T21:44:22.491+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-11T21:44:22.504+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-11T21:44:22.506+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
